c2corg_api/scripts/migration/init_feed.py

```diff
@@ -15,6 +15,7 @@ def migrate(self):
             self.session_target.execute(SQL_DOCUMENT_CREATE_CHANGE)
             self.session_target.execute(SQL_AREAS_FOR_CHANGES)
             self.session_target.execute(SQL_ACTIVITIES_FOR_CHANGES)
+            self.session_target.exececute(SQL_LANGUAGES_FOR_CHANGES)
```

Comment:
Ok, so we probably won't need this (IMO I would keep it in the code just for completeness).